mailpoet/lib/Doctrine/Middlewares/PostConnectMiddleware.php

```diff
@@ -0,0 +1,14 @@
+<?php declare(strict_types = 1);
+
+namespace MailPoet\Doctrine\Middlewares;
+
+use MailPoetVendor\Doctrine\DBAL\Driver\Connection;
+use MailPoetVendor\Doctrine\DBAL\Driver\Middleware\AbstractDriverMiddleware;
+
+class PostConnectMiddleware extends AbstractDriverMiddleware {
+  public function connect(array $params): Connection {
+    $connection = parent::connect($params);
+    $res = $connection->exec('SET time_zone = "+00:00"');
```

Comment:
`$res` seems to be unused (but no CI checks are complaining).

Reply:
You're right; I used it to debug the return value. But strange there are no warnings